res/controllers/Hercules_DJControl_Inpulse_300.midi.xml

```diff
@@ -248,14 +258,24 @@
 			<!--Q B-->
 			<control>
 				<group>[Channel2]</group>
-				<key>keylock</key>
+				<key>quantize</key>
 				<description>Q button</description>
 				<status>0x92</status>
 				<midino>0x02</midino>
 				<options>
 					<normal />
 				</options>
 			</control>
+			<control>
+				<group>[Channel2]</group>
+				<key>beats_translate_curpos</key>
+				<description>Shift + Q button</description>
+				<status>0x95</status>
+				<midino>0x02</midino>
+				<options>
```

Comment:
I guess all these `normal` options elements are redundant? Probably be generated by the mapping wizard? Doesn't really matter, though.

Reply:
I have no idea what these `options` do. I just copied code from other mappings and changed the values.

It seems that mapping input (LED) and output could've been done on the same place but I don't know if it's possible.